src/Core/Foundation/Version.php

```diff
@@ -162,14 +168,12 @@ public static function buildFromString($version)
      */
     public function getVersion($full = false)
     {
-        $version = '1.' . $this->version;
-
         if (!$full) {
             // remove extra parts
-            return preg_replace('/[-\+].*/', '', $version);
+            return preg_replace('/[-\+].*/', '', $this->version);
         }
 
-        return $version;
+        return $this->version;
```

Comment:
You're making `version` (the legacy result) the same as `semVersion`, which is a breaking change and might produce unforeseen consequences. This change is also not aligned with the method's comment.

Reply:
:+1: